src/cpp/utils.h

```diff
@@ -97,6 +101,18 @@ class Utils {
   */
   void meshgrid(const vec &x, const vec &y, const vec &z, cube &X, cube &Y,
                 cube &Z);
+
+ /**
+  * @brief Calculates the interior row for the kth order central finite difference scheme. This is in utils because 
+  *Both the gradient and interior operators need to use this for large k
```

Comment:
I think you need a space here to render this via Doxygen doc, i.e.,
```suggestion
  * Both the gradient and interior operators need to use this for large k
```